files/usr/lib/jeos-firstboot

```diff
@@ -325,14 +347,119 @@ for d in ${candidates[@]} ; do
         break # one is enough
 done
 
+if is_raspberry; then
```

Comment:
It would be useful to have this in a while loop which:

- Initially asks whether WiFi should be set up
- Checks for an active connection at the end and shows an error if it does not work
- Allows to reenter everything if the connection fails

Reply:
Indeed I missed the confirmation asking if you want or not to setup the wi-fi, the retries and tests for connection would be good to have, but all my tests (maybe because of my wireless card) didn't get me good results trying to figure out, there are hard to hardcode timeouts involved, not exactly neat to implement, so I will go with a simpler version for now and after defining some tests we can implement the retries.